Owncloud iOs Client/AppDelegate.m

```diff
@@ -674,14 +671,13 @@ + (OCCommunication*)sharedOCCommunication
 
 + (OCCommunication*)sharedOCCommunicationDownloadFolder {
     static OCCommunication* sharedOCCommunicationDownloadFolder = nil;
-    if (sharedOCCommunicationDownloadFolder == nil)
-    {
-        
+    if (sharedOCCommunicationDownloadFolder == nil) {
+
         NSURLSessionConfiguration *configurationDownload = nil;
         
-        if (IS_IOS8) {
-            configurationDownload = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:k_download_folder_session_name];
-        }else{
+        if (k_is_sso_active || !k_is_background_active) {
+            configurationDownload = [NSURLSessionConfiguration defaultSessionConfiguration];
+        } else {
             configurationDownload = [NSURLSessionConfiguration backgroundSessionConfiguration:k_download_folder_session_name];
```

Comment:
Use backgroundSessionConfigurationWithIdentifier: instead of backgroundSessionConfiguration that is deprecated.